src/containers/layout/sidebar/Sidebar.tsx

```diff
@@ -6,18 +6,29 @@ import List from '@mui/material/List'
 import ListItem from '@mui/material/ListItem'
 
 import HashLink from '~/components/hash-link/HashLink'
+import useBreakpoints from '~/hooks/use-breakpoints'
 
 import { styles } from '~/containers/layout/sidebar/Sidebar.styles'
 
 interface SidebarProps {
   onClose: () => void
   navigationItems: { route: string; path: string }[]
+  accountItems: { route: string; path: string }[]
```

Comment:
those items types are common for routes, create for them common interface

Reply:
fixed